packages/repo/src/types.ts

```diff
@@ -1,5 +1,6 @@
 import { z } from 'zod'
-import { schema as common, def as commonDef } from '@atproto/common'
+import { def as commonDef } from '@atproto/common-web'
+import { schema as common } from '@atproto/common'
```

Comment:
Another of the same TS error type as pino, fixed by using the source `def` import from `common-web` instead of the re-exported version from `common`.